proposals/dictionary-expressions.md

```diff
@@ -402,36 +316,152 @@ Synthesized types are free to employ any strategy they want to implement the req
+```
+
+Note: the element `KeyValuePair<K1,V1>` types need not be identical to the `KeyValuePair<K2,V2>` type of the destination dictionary type.  They simply must be convertible to the `V1 this[K1 key] { ... }` indexer provided by the dictionary.
+
+Yes.  These are legal: [LDM-2024-03-11](https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-03-11.md#conclusions)
+
+### Answered question 2
+
+Can you spread a *non dictionary type* when producing a dictionary type'd value.  For example:
+
+```c#
+Dictionary<string, int> nameToAge = ["mads": 21, .. existingListOfKVPS];
+``` 
+
+**Resolution:** *Spread elements* of key-value pair collections will be supported in dictionary expressions. [LDM-2024-03-11](https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-03-11.md#conclusions)
+
+### Answered question 3
+
+How far do we want to take this KeyValuePair representation of things? Do we allow *key value pair elements* when producing normal collections? For example, should the following be allowed:
+
+```c#
+List<KeyValuePair<string, int>> = ["mads": 21];
+```
+
+**Resolution:** *Key value pair elements* will be supported in collection expressions for collection types that have a key-value pair element type. [LDM-2024-03-11](https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-03-11.md#conclusions)
+
+### Answered question 4
+
+Dictionaries provide two ways of initializing their contents.  A restrictive `.Add`-oriented form that throws when a key is already present in the dictionary, and a permissive indexer-oriented form which does not.  The restrictive form is useful for catching mistakes ("oops, I didn't intend to add the same thing twice!"), but is limiting *especially* in the spread case.  For example:
+
+```c#
+Dictionary<string, Option> optionMap = [opt1Name: opt1Default, opt2Name: opt2Default, .. userProvidedOptions];
+```
+
+Or, conversely:
+
+```c#
+Dictionary<string, Option> optionMap = [.. Defaults.CoreOptions, feature1Name: feature1Override];
+```
+
+Which approach should we go with with our dictionary expressions? Options include:
+
+1. Purely restrictive.  All elements use `.Add` to be added to the list.  Note: types like `ConcurrentDictionary` would then not work, not without adding support with something like the `CollectionBuilderAttribute`.
+2. Purely permissive.  All elements are added using the indexer.  Perhaps with compiler warnings if the exact same key is given the same constant value twice.
+3. Perhaps a hybrid model.  `.Add` if only using `k:v` and switching to indexers if using spread elements.  There is deep potential for confusion here.
+
+**Resolution:** Use *indexer* as the lowering form. [LDM-2024-03-11](https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-03-11.md#conclusions)
+
+## Retracted Designs/Questions
 
-There is a subtle concern around the following interface destinations:
+### Question: Should `k:v` elements force dictionary semantics?
```

Comment:
No one is coming with this recommendation anymore.  And we made a firm LDM stance in the last meeting that `k:v` was fine for normal non-dicttionary types.  So this question is retracted.